cloudofficeprint/config/pdf.py

```diff
@@ -13,79 +13,110 @@ class PDFOptions:
 
     def __init__(
         self,
```

Comment:
add setters for new options and change back the constructor signature for backwards compatibility